.github/workflows/ci_macos.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
       - name: Build qBittorrent (Qt5)
         if: ${{ startsWith(matrix.qt_version, 5) }}
         run: |
-          CXXFLAGS="$CXXFLAGS -Werror -Wno-error=deprecated-declarations" \
+          CXXFLAGS="$CXXFLAGS -Wall -Wextra -Wpedantic -Wshadow -Werror -Wno-error=deprecated-declarations" \
```

Comment:
It is already done in build scripts. FYI
https://github.com/qbittorrent/qBittorrent/blob/8df80b67f9f469a3240856f326d04da0ea221b11/cmake/Modules/CommonConfig.cmake#L51-L63
https://github.com/qbittorrent/qBittorrent/blob/8df80b67f9f469a3240856f326d04da0ea221b11/unixconf.pri#L13

Reply:
Thanks. I have uploaded a new diff.